examples/phoneid_consent/consent_history.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import print_function
+from telesignenterprise.phoneid import PhoneIdClient
+
+customer_id = "Put your Customer ID between these quotes."
+api_key = "Put your API key between these quotes."
+
+phone_number = "Put the phone number you want to retrieve consent history for between these quotes."
+
+phoneid = PhoneIdClient(customer_id, api_key)
+
+response = phoneid.consent_history(phone_number)
+
+print(response.json)
+
+"""
```

Comment:
File comments are usually placed at the top of python scripts.

Reply:
Okay I've added the changes.